support binding the outgoing ip

satmd opened the pull request:
This merge adds support for setting the outgoing ip - and adds the example regex for inspircd

jlu5 commented on antissh.py:
```diff
@@ -24,11 +24,17 @@
 NICKNAME = config.get('host', 'nickname', fallback='antissh')
 MODES = config.get('host', 'modes', fallback='')
 KLINE_CMD_TEMPLATE = config.get('host', 'kline_cmd', fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  Please fix your SSH daemon and try again later.\r\n')
+OUTGOING_IPPORT = (config.get('target', 'outgoing_ip', fallback='::'), 0)
```
I would name this option something as simple as `bindhost`, and remove the port reference since it isn't really relevant

kaniini commented:
the inspircd regex seems to be the same?  that doesn't seem right...

satmd commented:
- changed naming of OUTGOING_IPPORT to BINDHOST - but keeping the ...,0 in the tuple
- changed regex example copy&paste mistake for the inspircd example
